dataland-frontend/src/services/ApiClients.ts

```diff
@@ -63,8 +132,9 @@ export class ApiClientProvider {
     return getUnifiedFrameworkDataControllerFromConfiguration(framework, configuration);
```

Comment:
The getUnifiedFrameworkDataControllerFromConfiguration() function should also be adapted to use the Axios interceptor for authentication.

Reply:
Done.

Reply:
ok